### ISSUE TITLE
Make scripts/decodecode work for powerpc

### ISSUE THREAD

**Issue by mpe** [linuxppc/issues#437]
When we oops we print an instruction dump:

```
[ 4681.238745] Instruction dump: 
[ 4681.238749] fbc1fff0 f821ffc1 7c7d1b78 7c9c2378 ebc30028 7fdff378 48000018 60000000  
[ 4681.238765] 60000000 ebff0008 7c3ef840 41820048 <815f0060> e93f0000 5529077c 7d295378  
[ 4681.238782] ---[ end trace 0000000000000000 ]--- 
```

Folks have various scripts to decode it (eg. [my version](https://github.com/mpe/misc-scripts/blob/master/ppc/ppc-disasm)).

But there is a script in the tree, `scripts/decodecode`, which is what other arches use. Currently it doesn't work with our oops output. We should fix that.

**Comment by chleroy** [comment]
According to Matthew, the only thing we have to do is:

```
diff --git a/arch/powerpc/kernel/process.c b/arch/powerpc/kernel/process.c
index c17c974e5723..b6dfc9d1c40d 100644
--- a/arch/powerpc/kernel/process.c
+++ b/arch/powerpc/kernel/process.c
@@ -1359,7 +1359,7 @@ static void show_instructions(struct pt_regs *regs)
 	unsigned long nip = regs->nip;
 	unsigned long pc = regs->nip - (NR_INSN_TO_PRINT * 3 / 4 * sizeof(int));
 
-	printk("Instruction dump:");
+	printk("Code:");
 
 	/*
 	 * If we were executing with the MMU off for instructions, adjust pc

```

Ref https://lore.kernel.org/linux-mm/Yzc8hJL6cqxXCKaJ@casper.infradead.org/

**Comment by mpe** [comment]
Series posted: http://patchwork.ozlabs.org/project/linuxppc-dev/list/?series=321548

**Comment by mpe** [comment]
Merged as torvalds/linux@d90bb7b4fdaff3f2fa68c7af85de2ce9e70189b1
